litert/vendors/qualcomm/core/utils/miscs.h

```diff
@@ -51,6 +52,9 @@ void ConvertDataFromUInt16toInt16(absl::Span<const std::uint16_t> src,
 void ConvertDataFromInt4ToInt8(const void* src, std::vector<std::int8_t>& dst,
                                size_t num_bytes);
 
+void ConvertDataFromInt2ToInt8(const void* src, std::vector<std::int8_t>& dst,
+                               size_t num_bytes);
```

Comment:
Nit: Make related parameters closer.
```suggestion
void ConvertDataFromInt2ToInt8(const void* src, size_t num_bytes, std::vector<std::int8_t>& dst);
```